robotbase/src/main/java/com/systemmeltdown/robot/subsystems/ClimbSubsystem.java

```diff
@@ -1,39 +1,159 @@
 package com.systemmeltdown.robot.subsystems;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
 import com.ctre.phoenix.sensors.PigeonIMU;
+import com.systemmeltdown.robotlib.subsystems.drive.SkidSteerDriveSubsystem.Configuration;
```

Comment:
What is the drive base configuration being used for in the climb sub?

Reply:
Good catch, thanks.